bigquery/main_test.go

```diff
@@ -142,18 +147,19 @@ func (bq *mockBQ) EnsureTable(ctx context.Context, tableName string) error {
 	}
 	if len(fakeResponse.table.Schema) == 0 {
 		bq.validSchema = true
-
 	}
+	bq.WrittenRows = []*bqRow{}
```

Comment:
I don't think you need to set this here. `list = append(nil, elem)` should work.